app/src/main/java/com/dodo/flashcards/domain/models/flashcard/IFlashcard.kt

```diff
@@ -0,0 +1,6 @@
+package com.dodo.flashcards.domain.models.flashcard
+
+interface IFlashcard {
```

Comment:
I know this is the C# pattern, but I would rename this just FlashCard to, the reason is that our view should be interfacing with the interface(which it is) and it looks a lot cleaner at least by kotlin style in the view layer to just say List<Flashcard>

The normal kotlin pattern is that your interface is named the theoretical thing you are creating then your implementing class is "thatthingImpl", so you'd have "Flashcard" and "FlashcardImpl"

Reply:
Oh yeah now I remember the impl - nice.